assets/js/blocks/cart-checkout/mini-cart/style.scss

```diff
@@ -76,6 +76,22 @@
 	}
 }
 
+// @todo Review the class naming convention for Mini Cart inner blocks.
+.wp-block-woocommerce-mini-cart-contents {
+	background: #fff;
+	box-sizing: border-box;
+	display: flex;
+	flex-direction: column;
+	height: 100%;
+	height: 100vh;
+	padding: $gap-largest $gap;
+}
+
+.wc-block-mini-cart__title {
+	font-size: xx-large;
```

Comment:
Ideally, we should always use one of the font-sizes from the mixin:

https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/ebae6b6da53924998d698991c5042148ad1b7484/assets/css/abstracts/_mixins.scss#L1-L13